spec/models/check_spec.rb

```diff
@@ -14,4 +14,12 @@
     expect(build(:check, result: "approved")).to be_valid
     expect(build(:check, result: nil)).not_to be_valid
   end
+
+  it "prevents an unapprovable claim from being approved" do
+    claim = create(:claim, :unverified)
+    check = build(:check, claim: claim, result: "approved")
+
+    expect(check).not_to be_valid
+    expect(check.errors.messages[:base]).to eq ["This claim cannot be approved"]
```

Comment:
Can we add brackets to the call to `eq` here? Seeing the start of the array threw me a bit.

Reply:
👍🏽